src/settings/s_decisions.php

```diff
@@ -147,7 +147,74 @@ function render(SettingValues $sv) {
         "<p class=\"settingtext\">To collect <em>multiple</em> final versions, such as one in 9pt and one in 11pt, add “Alternate final version” options via <a href='", hoturl("settings", "group=opt"), "'>Settings &gt; Submission options</a>.</p>",
         "</div>\n\n";
     Ht::stash_script("foldup(\$\$('cbfinal_open'),null,{f:\"c\"})");
+
+	//Share with Open Academic
+    echo "<h3 class=\"settings g\">Share with search engines</h3>\n";
+    echo "<p>This sharing service is hosted by <a href = 'http://openacademic.ai'>Open Academic Society</a> to to create a shared, open and expanding knowledge graph of research and education-focused entities and relationships.</p>";
+    echo "<p>It only shares <b>papers' metadata</b> but not the papers' actual content. For more information on the API, please click on link: <a href = 'https://api.openacademic.ai/Help'>Open Academic API</a></p>";
+    echo "<p>Please select the corresponding <b>acceptance</b> type you want to share: ".decisionSelector(1)."</p>";
+    echo "<p>Please insert the <b>authentication token</b> acquired from <a href = 'https://api.openacademic.ai/tokenrequest'>Open Academic Authentication</a>: ".Ht::entry("token", "");
+    echo "<p><b>Click the button to share accepted paper meta data with search engines</b></p>";
+    echo Ht::submit("share", "Share");
+   
+    $api = "https://api.openacademic.ai/api/papers";
+    if (isset($_REQUEST["share"])) {
+        $this->share($sv, $api);
+    }
+
 }
+    //sends https post request to the api
+    function share(SettingValues $sv, $api) {
+        $decision = $_REQUEST["decision"];
+
+        $share = array();
+        $share["provider"] = "HotCRP";
+        $share["setId"] = "";
+        $share["venue"] = $sv->conf-> long_name;
+        $share["shortVenue"] = $sv->conf-> short_name;
+        $share["siteUrl"] = $sv->conf-> opt("conferenceSite");
+    
+        $papers = array();
+        $result = $sv->conf-> q_raw("SELECT * FROM Paper WHERE outcome = $decision");
```

Comment:
this is a SQL injection attack

$sv->conf->q("select * from Paper where outcome=?", $decision)